### ISSUE TITLE
Progress logging to stderr 

### ISSUE THREAD

**Issue by divya1c** [mapbox/tippecanoe#428]
Hi! 
I run this command and I find that all the progress log goes into stderr.

```tippecanoe -X -f -o points.mbtiles -b0 -r1 -pk -pf -l "avg" -z22 -Z15 points.geojson 1>stdout.txt 2>stderr.txt```

It makes it difficult for me to track errors from tippecanoe by parsing my log. I am aware that I can quiet the progress log. But, I want to also keep the warning log. Also, is there a way to quiet the log alone? 

Thanks :)

**Comment by ericfischer** [comment]
`-q` should quiet the warnings but it will also eliminate the progress indicator. It would probably be reasonable to have separate `--quiet-warnings` and `--quiet-progress` flags if that would help.

I think it is a standard convention for progress indicators to write to stderr.  (Compare, for example, to `curl`, which does the same thing.) Were you expecting to see it on stdout?

**Comment by divya1c** [comment]
yeah, having different flags would help :) I probably will keep the warnings and quiet the log, for testing purposes. 

Also, I was thinking - do you think a shorter progress log would help a lot for non-interactive streams? I may not want to see a lot of progress log when I redirect it into a file for logging purposes (it creates a lot of clutter), but I may want to see it when its printing out to my interactive console.

**Comment by divya1c** [comment]
Also, when I do -q, I still see this in my stderr log : 
```
points.geojson: Warning: GeoJSON specified projection "urn:ogc:def:crs:EPSG::4269", not the expected "urn:ogc:def:crs:OGC:1.3:CRS84".
points.geojson: If "urn:ogc:def:crs:OGC:1.3:CRS84" is not the expected projection, use -s to specify the right one.
```

Isn't this a warning and should be quiet?

**Comment by ericfischer** [comment]
Oh, good point, `-q` ought to affect the projection warning.

The progress indicator is probably noisier than it really ought to be. Currently it reports every 0.1% during tiling, but it probably should be limited to report only once per second or something like that.

**Comment by ericfischer** [comment]
Warning fixed in 0801a9324be12d
